noxfile.py

```diff
@@ -26,3 +30,11 @@ def run_template_generation(session):
     session.install("setuptools", silent=False)
     session.install("-e", ".[dev]", silent=False)
     session.run("pytest", "tests")
+
+@nox.session(name="dev")
```

Comment:
```suggestion
@nox.session(name="dev", reuse_venv=True)
```
I think it should be good to have `reuse_venv=True` in sessions so feel free to do that for others too

Reply:
We already have `nox.options.reuse_existing_virtualenvs = True` at the top as a global setting so this argument would not be needed